syntax = "proto2";

package echo;

message RpcRequest
{
	required string message = 1;
};

message RpcResponse
{
	required string response = 1;
};

service EchoService
{
	rpc Echo(RpcRequest) returns (RpcResponse);
};

option cc_generic_services = true;